anitya/lib/backends/rubygems.py

```diff
@@ -74,3 +74,27 @@ def get_versions(cls, project):
                 'Project or version unknown at %s' % url)
 
         return [data['version']]
+
+    @classmethod
+    def check_feed(cls):
+        ''' Return a generator over the latest 50 uploads to rubygems.org
+
```

Comment:
The *best* case we could get would be to have all of the upstream forges enabled with *webhooks*, so they can ping us every time someone uploads a new release of anything.

I looked at all of them last night, and I think rubygems.org is the only one [that can do this](http://guides.rubygems.org/rubygems-org-api/#webhook-methods).  If more of them supported it, it would be worth adding support to anitya to do it, I think.  Then we can have real-real-time.